docs/socialaccount/providers/saml.rst

```diff
@@ -73,9 +73,9 @@ via the Django admin as well:
                         # If left empty, an attempt will be done to map the attributes using
                         # built-in defaults.
                         "attribute_mapping": {
-                            "uid": "http://schemas.auth0.com/clientID",
-                            "email_verified": "http://schemas.auth0.com/email_verified",
-                            "email": "http://schemas.xmlsoap.org/ws/2005/05/identity/claims/emailaddress",
+                            "uid": "urn:oid:1.3.6.1.4.1.5923.1.1.1.6",
+                            "email_verified": "true",
```

Comment:
`"true"` seems wrong here, that is not an attribute?

Reply:
This is interesting: everything I've been able to find says this will vary by IdP, and some places have suggested defaulting to using the same field as email. The IdP I'm working with (University of PA) only supports `urn:oid`. I initially had `urn:oid:0.9.2342.19200300.100.1.3` (email) in as the default for this value as well. Do you think that would be a good default? I swtiched to `"true"` after examining the code in the provider checking attribute values.

Reply:
After speaking with our identity, team, they recommended using the said `urn:oid` value for `email_verified` as a default, since there's no definition in the schema.

Reply:
If there is nothing representing `email_verified` then we shouldn't be listing any `urn` value at all.